specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -4298,6 +4298,58 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/sqlDatabases/{databaseName}/containers/{containerName}/retrieveContinuousBackupInformation": {
+      "post": {
```

Comment:
We usually use POST to perform an action on the resource. I don't know the scenario here, but it looks like we are doing a read here.

Reply:
It is actually kind of action. The 'retrieveContinuousBackupInformation' action is going to trigger async workflow on backend to retrieve all the backup information